### PR TITLE
Name polish + add nativeCoinName

### PR DESCRIPTION
**Pull request by hpihkala**
The frontend is using the `name` fields, so make them more human-readable. The name of the native coin on each chain is also needed.

@samt1803 if you're happy with the changes, could you please merge the changes and publish a new config version for the frontend?

### CODE REVIEW

**Comment by tumppi** [comment]
Hey, I noticed that we need to provide following object to `wallet_addEthereumChain` (https://eips.ethereum.org/EIPS/eip-3085) on the frontend:
```
nativeCurrency?: {
  name: string;
  symbol: string;
  decimals: number;
};
```

> The wallet MUST reject the request if the nativeCurrency field is provided, and any of the name, symbol, or decimals fields are missing. The wallet MUST reject the request if the decimals field is a negative integer

**Comment by jtakalai** [comment]
> Or should we force 18 decimals (which should be the case for EVM-based chains) for all and use `name === symbol`.
For operator tokens, this sounds completely correct. For native tokens... maybe consult an existing list such as https://github.com/viaprotocol/tokenlists/tree/main/all_tokens

**Comment by jtakalai** [comment]
I think I'll dig up the native token names too from that list.

File: packages/config/src/index.ts [hunk 1]
```diff
@@ -248,7 +293,11 @@ export const config = {
     },
     "mumbai": {
         "id": 80001,
-        "name": "mumbai",
```
Comment (tumppi) [inline packages/config/src/index.ts:251]: Name removed by accident?

File: packages/config/src/index.ts [hunk 2]
```diff
@@ -248,7 +293,11 @@ export const config = {
...
+        "nativeCoin": {
+            "symbol": "MATIC",
+            "name": "Mumbai",
+            "logoURI": "https://tokens.1inch.io/0x7d1afa7b718fb893db30a3abc0cfc608aacfebb0.png"
```
Comment (tumppi) [inline packages/config/src/index.ts:299]: We cannot provide `logoURI` to MetaMask so we might as well remove the logo altogether?

**Comment by hpihkala** [comment]
@tumppi take this over to provide what you need? Also `TEST` might be a bad choice of name for the native coin of the local chains, since we also use a `TEST` token.

**Comment by tumppi** [comment]
@samt1803 This should now have everything we need for frontend. Please let me know when a new npm package version is available.

**Review by samt1803 — APPROVED** [review]

**Pull request merged**